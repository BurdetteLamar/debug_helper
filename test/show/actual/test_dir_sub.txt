---
DebugHelperTest::DirSub (message='test_dir_sub'):
  DebugHelperTest::DirSub#path: C:/Users/Burde/Documents/GitHub/debug_helper/test
  DebugHelperTest::DirSub.entries("C:/Users/Burde/Documents/GitHub/debug_helper/test"):
  - "."
  - ".."
  - debug_helper_test.rb
  - show
  - show_hash
  - test_helper.rb
  DebugHelperTest::DirSub.exist?("C:/Users/Burde/Documents/GitHub/debug_helper/test"): true
