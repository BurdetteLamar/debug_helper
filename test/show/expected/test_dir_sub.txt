---
DebugHelperTest::DirSub (message='test_dir_sub'):
  DebugHelperTest::DirSub#path: git_dir/test
  DebugHelperTest::DirSub.entries("git_dir/test"):
  - "."
  - ".."
  - debug_helper_test.rb
  - show
  - show_hash
  - test_helper.rb
  DebugHelperTest::DirSub.exist?("git_dir/test"): true
